controllers/tasks.js

```diff
@@ -17,8 +18,12 @@ const userQuery = require("../models/users");
  */
 const addNewTask = async (req, res) => {
   try {
+    // userStatusFlag is the Feature flag for status update based on task status. This flag is temporary and will be removed once the feature becomes stable.
+    const { userStatusFlag } = req.query;
```

Comment:
Are we using dev query here or leveraging the feature flag service? Is the `true` denoting the `dev=true` feature in the UI?

Reply:
we are using the feature flag service. As the dev = true was already used. So had to create a new one. In the UI Changes are yet to be implemented.